elyra/pipeline/processor_local.py

```diff
@@ -57,6 +60,17 @@ def __init__(self, root_dir, **kwargs):
     def type(self):
         return self._type
 
+    @property
+    def registry_location(self) -> str:
```

Comment:
This property and `component_parser` should be removed and these should be defined on `RuntimePipelineProcessor` since they don't apply to local processors.  Then the constructors for `AirflowPipelineProcessor` and `KfpPipelineProcessor` should be refactored with most all of their content pushed to the (new) `RuntimePipelineProcessor` constructor.

I'm happy to add that commit if you like.